tests/unit/test_webhook.py

```diff
@@ -128,6 +128,20 @@ def setUp(self):
         **Tested Another Browser**: No
         """  # noqa
 
+        self.issue_body10 = """
+        <!-- @browser: Firefox iOS 34.1 -->
+        <!-- @ua_header: Mozilla/5.0 (iPad; CPU iPhone OS 14_5 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/33.1  Mobile/15E148 Safari/605.1.15 -->
```

Comment:
Maybe it's worth adding another test, that has desktop Safari UA (default UA of iPad on Firefox at the moment).
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1 Safari/605.1.15`
This is majority of the reports from an iPad right now

Issues will be coming with the actual iPad UA only if mobile site is requested explicitly and it's very rare.

Reply:
That's a good point, I will add this. Thanks Ksenia.

Reply:
https://github.com/webcompat/webcompat-tests/issues/2658

Reply:
https://github.com/webcompat/webcompat.com/pull/3584/commits/f11749c59d69acfd2d80a2f3ebaa032ab06caacb